src/screens/SeekHelp/index.tsx

```diff
@@ -1,25 +1,179 @@
+        return prevSelected.filter(item => item !== level);
+      } else {
+        return [...prevSelected, level];
+      }
+    });
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const fetchData = async () => {
+    try {
+      const data = await getSeekHelpData();
+      setResources(data);
+      setFilteredResources(data);
+    } catch (error) {
+      console.error('Error fetching data:', error);
+    }
+  };
+
+  const applyFilter = (filter: string) => {
+    setSelectedFilter(filter);
+    const tags = tagMapping[filter as keyof typeof tagMapping];
+    // iterates through resources to check if the tags are equal to the filter
+    const filtered = resources.filter(resource =>
+      resource.tags.split(',').some((tag: string) => tags.includes(tag.trim())),
+    );
+    setFilteredResources(filtered);
+  };
+
+  const applyJurisdictionFilter = () => {
+    let filtered = resources;
+
+    if (selectedJurisdiction.length > 0) {
+      filtered = filtered.filter(resource => {
+        // check if resource state matches any selected jurisdiction
+        return selectedJurisdiction.includes(resource.state);
+      });
+    }
+
+    if (selectedFilter) {
+      const tags = tagMapping[selectedFilter as keyof typeof tagMapping];
+      filtered = filtered.filter(resource =>
+        resource.tags
+          .split(',')
+          .some((tag: string) => tags.includes(tag.trim())),
+      );
+    }
+
+    setFilteredResources(filtered);
+  };
+
+  useEffect(() => {
+    applyJurisdictionFilter();
+  }, [selectedJurisdiction, selectedFilter, resources]);
+
   return (
     <View style={styles.container}>
-      <TouchableOpacity
-        style={styles.pagebutton}
-        onPress={() => navigation.navigate('ResourceList')}
-      >
-        <Text style={styles.buttonText}>California</Text>
-      </TouchableOpacity>
-      <TouchableOpacity
-        style={styles.pagebutton}
-        onPress={() => navigation.navigate('ResourceList')}
-      >
-        <Text style={styles.buttonText}>National</Text>
-      </TouchableOpacity>
+      <View style={styles.leftPanel}>
+        <View style={styles.selectContainer}>
+          <Text style={styles.selectText}>Select Resources Level</Text>
+          <View style={styles.selectJurisdictionContainer}>
+            {jurisdiction.map(level => (
+              <TouchableOpacity
+                key={level}
+                style={styles.checkButtonContainer}
+                onPress={() => toggleSelection(level)}
+              >
+                <View
+                  style={[
+                    styles.checkButton,
+                    selectedJurisdiction.includes(level)
+                      ? styles.selectedCheckButton
+                      : null,
+                  ]}
+                >
+                  {selectedJurisdiction.includes(level) && <Check />}
+                </View>
+                <Text style={styles.buttonText}>{level}</Text>
+              </TouchableOpacity>
```

Comment:
I changed it for you - but I basically swapped the inner view and touchable opacity. generally we want buttons to take up as much space as possible in the container such that things are more clickable (and thereby accessible) - this is why I try and refrain from using margins and normally use padding! You can see that adding padding or simply swapping react components around can make the clickable area a lot bigger and more intuitive to use.